http/src/main/resources/reference.conf

```diff
@@ -369,6 +369,12 @@ zombieRuntimeMonitor {
   concurrency = 100
 }
 
+persistentDisk {
+  defaultDiskSizeGB = 500
+  defaultDiskType = "pd-standard"
+  defaultBlockSizeBytes = 32 # TODO: figure out what this really should be
```

Comment:
From docs the allowed values are 4096 and 16384. Probably use 4096 as a default.

https://cloud.google.com/compute/docs/reference/rest/v1/disks/insert